Source/scrollrt.cpp

```diff
@@ -188,17 +188,13 @@ static void scrollrt_draw_cursor_item(const CelOutputBuffer &out)
 
 	const auto &sprite = GetInvItemSprite(pcurs);
 	const int frame = GetInvItemFrame(pcurs);
+	void (*drawingFun)(const CelOutputBuffer &, int, int, const CelSprite &, int) = &CelClippedDrawSafeTo;
 	if (pcurs >= CURSOR_FIRSTITEM) {
 		const auto &heldItem = plr[myplr].HoldItem;

```

Comment:
If you follow my wrapper function suggestion, this would become:

```c++
if (pcurs >= CURSOR_FIRSTITEM) {
	const auto &heldItem = plr[myplr].HoldItem;
	CelBlitOutlineTo(out, GetOutlineColor(heldItem, true), MouseX, MouseY + cursH - 1, sprite, frame, false);
}
CelDrawWrapper(!heldItem._iStatFlag, out, MouseX, MouseY + cursH - 1, sprite, frame);
```